src/contact-summary/contact-summary-emitter.js

```diff
@@ -27,6 +27,11 @@ function emitter(contactSummary, contact, reports) {
     var idx1, r, added;
 
     var appliesToType = convertToArray(card.appliesToType);
+
+    var appliesToTypeWithoutReport = appliesToType.filter(function(t) { return t !== 'report'; });
+    if (appliesToType.includes('report') && appliesToTypeWithoutReport.length > 0) {
```

Comment:
what about just using `appliesToType.includes('report') && appliesToType.length > 1`? seems unlike anybody would put report twice and throwing in that event is consistent with your error message